cilium/cmd/monitor.go

```diff
@@ -113,6 +115,7 @@ func init() {
 	monitorCmd.Flags().Var(&toDst, "to", "Filter by destination endpoint id")
 	monitorCmd.Flags().Var(&related, "related-to", "Filter by either source or destination endpoint id")
 	monitorCmd.Flags().BoolVarP(&verboseMonitor, "verbose", "v", false, "Enable verbose output")
+	monitorCmd.Flags().BoolVarP(&jsonOutput, "json", "j", false, "Enable json output. Shadows -v flag")
```

Comment:
Shouldn't we use `command.AddJSONOutput(monitorCmd)` instead, like in the other commands?

https://github.com/cilium/cilium/blob/master/cilium/cmd/bpf_ct_list.go#L49
https://github.com/cilium/cilium/blob/master/cilium/cmd/bpf_ct_list.go#L64-L65

https://github.com/cilium/cilium/blob/master/cilium/cmd/status.go#L57
https://github.com/cilium/cilium/blob/master/cilium/cmd/status.go#L74-L77

By implementing JSON output in that way, the command supports `-o json` option.

Refactoring of all commands in that way is tracked in https://github.com/cilium/cilium/issues/2083

Reply:
This is tempting, but `command.AddJSONOutput` adds `jsonpath` as possible value of `-o` option, which doesn't make any sense in monitor context, which outputs multiple self-contained json objects (one per line).

I am willing to make the monitor option look as the other subcommands, so will change the flag format to `-o`.

Reply:
Yes, sounds good, I didn't notice the other modes.